plugin/pom.xml

```diff
@@ -34,7 +34,7 @@
     -->
     <hpi.compatibleSinceVersion>1.28</hpi.compatibleSinceVersion>
     <hpi.pluginChangelogUrl>https://github.com/jenkinsci/configuration-as-code-plugin/releases</hpi.pluginChangelogUrl>
-    <hpi.pluginLogoUrl>https://raw.githubusercontent.com/jenkinsci/configuration-as-code-plugin/master/plugin/src/main/webapp/img/logo-head.svg</hpi.pluginLogoUrl>
+    <hpi.pluginLogoUrl>https://raw.githubusercontent.com/jenkinsci/configuration-as-code-plugin/master/plugin/src/main/resources/images/symbols/logo.svg</hpi.pluginLogoUrl>
```

Comment:
interesting I don't think this is actually used anywhere tbh

Reply:
No idea. It's the first time I see a pom with `hpi.pluginLogoUrl`

Reply:
2 plugins have it according to:
https://github.com/search?q=org%3Ajenkinsci%20hpi.pluginLogoUrl&type=code

but it appears to be unused anywhere, I think it was unfinished work to maybe show in plugin manager or plugins.jenkins.io